.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "mdf"]
+	path = mdf
+	url = https://github.com/edwardcwang/plsi-mdf.git
```

Comment:
@edwardcwang I didn't understand you mean a submodule by a library. We should avoid it.